packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -62,11 +70,53 @@ export async function processOutboundPatientDiscoveryResps({
       },
     });
 
-    log(`Completed.`);
+    const rerunPdOnNewDemographics = cqData?.rerunPdOnNewDemographics;
```

Comment:
can you write out briefly what the new flow and logic is here? Just trying to understand better

Reply:
The idea is as follows:
1. If the current run of PD has `rerunPdOnNewDemographics` true, check to see if there's new demographics.
2. Prep the next scheduled PD run (if any).
3. If there were new demographics from (1), then *override* the next scheduled PD run (if any) with a new `augmentDemographics = true, rerunPdOnNewDemographics = false` run flagged with `isRerunFromNewDemographics` and set this to the next scheduled PD run. If there were not new demographics, simply leave the prepped scheduled PD run (if any) from (2).
4. Process the scheduled PD run (if any) -- if there is a next scheduled PD run, run it and reset the schedule. If the scheduled PD run is flagged with `isRerunFromNewDemographics` and the current run has `rerunPdOnNewDemographics = true`, then we're in the "re-run with no DQ state" and skip DQ.
5. If no scheduled PD run, set status to completed and fire off DQ.

Reply:
Will clarify after standup.

Reply:
Irrelevant now: flow has been updated to de-couple rerun and scheduling.